src/tests/enforce-boolean-naming-prefixes.test.ts

```
@@ -0,0 +1,305 @@
+import { ruleTesterTs } from '../utils/ruleTester';
+import { enforceBooleanNamingPrefixes } from '../rules/enforce-boolean-naming-prefixes';
+
+ruleTesterTs.run('enforce-boolean-naming-prefixes', enforceBooleanNamingPrefixes, {
```

ESLint

[error] Replace `'enforce-boolean-naming-prefixes',·enforceBooleanNamingPrefixes,` with `⏎··'enforce-boolean-naming-prefixes',⏎··enforceBooleanNamingPrefixes,⏎·`

(prettier/prettier)